### PR TITLE
postprocess: don't ratelimit kmsg in initramfs

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

**Comment by dustymabe** [comment]
reworking this a bit

**Comment by dustymabe** [comment]
we should be good now.. had to track down a few things. See the comments inline in the text for good information on why this is needed.

File: fedora-coreos-base.yaml [hunk 1]
```diff
@@ -146,6 +146,31 @@ postprocess:
...
+    cat << 'EOF' > /usr/lib/dracut/modules.d/10coreos-sysctl/module-setup.sh
+    #!/usr/bin/bash
+    check() { return 0; } # include by default
+    install() {
+        mkdir -p "$initdir/etc/sysctl.d"
+        echo "kernel.printk_devkmsg = on" > "$initdir/etc/sysctl.d/10-dont-ratelimit-kmsg.conf"
+    }
+    EOF
+    # After initramfs (in booted system), switch back to ratelimit
+    cat << 'EOF' > /etc/sysctl.d/10-coreos-ratelimit-kmsg.conf
```
Comment (bgilbert) [inline fedora-coreos-base.yaml]: Should this be in `/usr/lib/sysctl.d` instead?

Comment (dustymabe) [reply fedora-coreos-base.yaml]: 👍 - fixed

**Pull request merged**